android/build.gradle

```diff
@@ -9,6 +9,10 @@ buildscript {
   }
 }
 
+task prepareKotlinBuildScriptModel {
```

Comment:
why is this task added explicitly here?

Reply:
Gradle Sync executes `:prepareKotlinBuildScriptModel` which runs all `prepareKotlinBuildScriptModel` tasks in a project. Often there's only `:app:prepareKotlinBuildScriptModel` but here we register an empty task `:expensify_react-native-live-markdown:prepareKotlinBuildScriptModel` so it runs during Gradle Sync.